web/blog/vast-v2.3/index.md

```diff
@@ -0,0 +1,55 @@
+---
+draft: true
+title: VAST v2.3
+description: Automatic Rebuilds
+authors: dominiklohmann
+date: 2022-08-05
+tags: [release, rebuild, performance]
+---
+
+We released [VAST v2.3][github-vast-release]! 
+
+FIXME: Write a short summary.
+
+[github-vast-release]: https://github.com/tenzir/vast/releases/tag/v2.3.0
+
+<!--truncate-->
+
+## Automatic Rebuilds
+
+VAST server processes now continually rebuild outdated and merge undersized
+partitions in the background. To control this behavior, set the new
+`vast.automatic-rebuild` option.
+
+```yaml
+vast:
+  # Control automatic rebuilding of partitions in the background for
+  # optimization purposes. The given number controls how many rebuilds to run 
+  # concurrently, and thus directly controls the performance vs. memory and CPU
+  # usage trade-off. Set to 0 to disable. Defaults to 1.
+  automatic-rebuild: 1
+```
+
+This has two major advantages:
+
+1. For VAST users, optimally sized partitions in the most recent will be the
+   norm. This improves query performance throughout the board.
+
+2. For VAST developers, making changes to VAST's partitions has less of a
+   barrier to it, because users by default update to the most recent veersion
+   over time. This makes development on one of the core pieces of VAST simpler.
```

Comment:
I struggle with understanding this. What is the actual *tangible* benefit?

Reply:
I rewrote it like this:

> 2. For VAST developers, making changes to VAST's partitions has less of a barrier to it, because they need to worry less over backwards compatibility as VAST now ensures partitions are up to date. This will allow for removing support for partitions stored in older formats in future major releases of VAST.

Is that clearer? Open to suggestions, my brain is not good at formulating things clearly with today's heat.

Reply:
I think that's clearer. Here's an attempt to simplify:

> Automatic also rebuilding helps with keeping partitions at the latest version. This make it faster to adopt VAST version that include breaking changes in the storage layout.

Note: I don't think this applies to developers only.

Reply:
Great, thanks for the suggestion. I applied it with an additional typo fix.